configs/_base_/datasets/s3dis-3d-5class.py

```diff
@@ -0,0 +1,113 @@
+# dataset settings
+dataset_type = 'S3DISDataset'
+data_root = './data/s3dis/'
+class_names = ('table', 'chair', 'sofa', 'bookcase', 'board')
+train_area = [1, 2, 3, 4, 6]
+test_area = 5
+
+train_pipeline = [
+    dict(
+        type='LoadPointsFromFile',
+        coord_type='DEPTH',
+        shift_height=True,
+        load_dim=6,
+        use_dim=[0, 1, 2, 3, 4, 5]),
+    dict(type='LoadAnnotations3D', with_bbox_3d=True, with_label_3d=True),
+    dict(type='IndoorPointSample', num_points=40000),
```

Comment:
In PR #667, we unify indoor and outdoor point sampling with a transform `PointSample`. Please kindly change the name.

Reply:
See this [line](https://github.com/open-mmlab/mmdetection3d/pull/667/files#diff-2d0d31184b759c5edf7a417f47c6994da36a8d640ce4d79a334179f98fa5d003R23)